homeassistant/components/mobile_app/webhook.py

```diff
@@ -48,14 +48,26 @@ def register_deleted_webhooks(hass: HomeAssistantType, store: Store):
             pass
 
 
-def setup_registration(hass: HomeAssistantType, store: Store,
-                       registration: Dict) -> None:
+async def setup_registration(hass: HomeAssistantType, store: Store,
+                             registration: Dict) -> None:
     """Register the webhook for a registration and loads the app component."""
     registration_name = 'Mobile App: {}'.format(registration[ATTR_DEVICE_NAME])
     webhook_id = registration[CONF_WEBHOOK_ID]
     webhook_register(hass, DOMAIN, registration_name, webhook_id,
                      partial(handle_webhook, store))
 
+    device_registry = await dr.async_get_registry(hass)
+    device_registry.async_get_or_create(
+        config_entry_id=webhook_id,
+        identifiers={
+            (DOMAIN, webhook_id),
```

Comment:
Use registration ID

Reply:
Registration ID is implemented in another PR (#21849), wat do?

Reply:
Wait for that PR ;)